ArduPlane/commands_logic.cpp

```diff
@@ -348,6 +348,9 @@ void Plane::do_takeoff(const AP_Mission::Mission_Command& cmd)
     steer_state.locked_course_err = 0;
     steer_state.hold_course_cd = -1;
     auto_state.baro_takeoff_alt = barometer.get_altitude();
+    if (!plane.ahrs.airspeed_sensor_enabled() && plane.g2.flight_options & FlightOptions::SURPRESS_TKOFF_SCALING) {
```

Comment:
interesting that you are only applying this in takeoff. From your description I thought you wanted this all the time.
If done in just takeoff then you don't need the extra variable, you can just check 
  flight_stage == AP_Vehicle::FixedWing::FLIGHT_TAKEOFF
or perhaps we should do it based on when is_flying changed? ie. apply it millis()-started_flying_ms < 20000 to apply to the first 20s of flight in all modes?

Reply:
if its only for the climb portions of mode TAKEOFF and NAV_TAKEOFF, its would be what I desired ...I think an arbitrary time may be an issue in a long climb in strong headwinds and be counterproductive for fast climbers with short climb periods...will change to flight stage check...thanks